component/provider.jsonnet

```diff
@@ -304,4 +304,56 @@ local controllerConfigRef(config) =
       role,
       rolebinding,
     ],
+  [if params.providers.minio.enabled then '10_provider_minio']:
```

Comment:
If I'm not wrong we decided to include everything in composition functions.

Reply:
The provider itself is not deployed by a composition. It's deployed like all other providers via the Crossplane package.

Reply:
Ah I missed that. Good then.